replication-api-impl/src/main/java/com/connexta/ion/replication/api/impl/ReplicatorImpl.java

```diff
@@ -166,21 +166,21 @@ private void completeActiveSyncRequest(SyncRequest syncRequest) {
   public void cleanUp() {
     final RetryPolicy retryPolicy =
         new RetryPolicy()
-            .retryWhen(false)
-            .withDelay(1, TimeUnit.SECONDS)
-            .withMaxDuration(30, TimeUnit.SECONDS);
+            .onRetry(
```

Comment:
✏️ We should only be adding this `onRetry()` if debug is enabled to prevent failsafe from calling this method each time.
✏️ Might also be more readable to create a private method named `private void logNumberOfPendingAndActiveSyncRequests(XXX isEmpty)` and simply replace this line with `.onRetry(this::logNumberOfPendingAndActiveSyncRequests)`